robocorp-code/vscode-client/src/extensionCreateEnv.ts

```diff
@@ -207,6 +278,13 @@ export async function createDefaultEnv(
         return;
     }
 
+    const convertBundleLocation = await ensureConvertBundle();
```

Comment:
We shouldn't do this here. We only want to download when the action is actually executed.

Reply:
This is where I figured we should download the bundle just once when the Extension is initialised & afterwards check the existence of the file.
Probably I got confused between these two comments: https://github.com/robocorp/robotframework-lsp/pull/757#discussion_r972272701

As the action has additional steps, it might be a bit tedious to wait for download to finish and move on. 
It seemed natural to do it in the env init as it is part of the environment functionality. 
But if you think I should move this, wouldn't be a problem on my end 😊

Reply:
The idea here is that most users will almost never use this feature, so, there's no point to everyone paying for that cost at activation time.

Also, we want to update the bundle when a new one comes out (but we should just do the update check once per session, so, just recheck for a new version after a VSCode restart).